src/mobot_client/admin.py

```diff
@@ -42,19 +61,30 @@ class CustomerDropRefundsAdmin(admin.ModelAdmin):
 
 
 class DropSessionAdmin(admin.ModelAdmin):
-    pass
+    list_display = ('drop', 'customer', 'created_at', 'state', 'updated')
+
+    @admin.display(description='Drop')
+    def drop(self, obj):
+        return obj.drop.name
 
 
 class MessageAdmin(admin.ModelAdmin):
     pass
 
 
 class BonusCoinAdmin(admin.ModelAdmin):
-    pass
+    list_display = ('__str__', 'number_remaining', 'amount_mob')
+    readonly_fields = ('number_claimed', 'number_remaining',)
 
+    @admin.display(description='MOB')
+    def amount_mob(self, obj):
+        return f"{pmob2mob(obj.amount_pmob):.4f}"
 
 class SkuAdmin(admin.ModelAdmin):
-    pass
+    readonly_fields = ('number_available',)
+
+    def get_queryset(self, request):
+        return self.model.objects.all()
```

Comment:
Is this still needed after the managers/queryset change?

Reply:
Yes, as I'm not changing Sku in this PR - just the DropSessions. I'll move SKU when I'm back to working on item drops.